docs/spec/sources.md

```diff
@@ -362,5 +370,10 @@ _Structured Content Mode_ of the HTTP Protocol Binding for CloudEvents. Sources
 SHOULD send events to its
 [Destination](https://pkg.go.dev/github.com/knative/pkg/apis/duck/v1?tab=doc#Destination).
 
+A source that generates or forwards CloudEvents to its destination MUST produce
+or extend (respectively) the CloudEvents with the `knsource` extension, with
+value following the format `plural.group/name`. Off cluster sources MAY produce
```

Comment:
Could you clarify what `plural.group/name` means? Without expansion I am not sure what is expected as the value.